pkg/alertmanager/amcfg_test.go

```diff
@@ -3789,7 +3329,7 @@ templates: []
 
 	for _, tc := range testCase {
 		t.Run(tc.name, func(t *testing.T) {
-			ac, err := alertmanagerConfigFromBytes(tc.rawConf)
+			ac, err := alertmanagerConfigFromBytes(golden.Get(t, tc.golden))
```

Comment:
We can't use golden.Get(), see comment in https://github.com/prometheus-operator/prometheus-operator/pull/5785#issuecomment-1670193826